runtime/doc/options.txt

```diff
@@ -7594,9 +7438,10 @@ A jump table for the options with a short description can be found at |Q_op|.
 	but the Windows version of Vim can source unix format scripts.
 
 				*'viminfo'* *'vi'* *E526* *E527* *E528*
-'viminfo' 'vi'		string	(Vi default: "", Vim default for MS-DOS,
-				   Windows: '100,<50,s10,h,rA:,rB:,
-				   for others: '100,<50,s10,h)
+'viminfo' 'vi'		string	(Vim default for
+				   Win32: "'100,<50,s10,h,rA:,rB:",
+				   others: "'100,<50,s10,h",
```

Comment:
> (looks correct in Vim, not on github)

I'm not sure what you intended it to look like as it seems that GH has converted all tabs to spaces.

> I think alignment could be used to make it more readable (removing " and terminating ,):

I pushed a commit doing that. I'm fine with removing the comma and aligning it, but I didn't want to remove the quotes because it breaks consistency for representing empty strings, like in your example. I'd prefer having quotes everywhere as opposed to replacing `""` with `(empty string)`, as the meaning is clearer IMO.